packages/frontend/components/primitives/tooltip/styles.ts

```diff
@@ -0,0 +1,17 @@
+﻿import { css } from '@emotion/react';
+import type { Theme } from 'theme';
+
+export const tooltipTrigger = () => css`
+  cursor: pointer;
+`;
+
+export const tooltip = ({ colors, spaces }: Theme) => css`
+  background-color: ${colors.background} !important;
+  border-radius: ${spaces.xs} !important;
+  padding: ${spaces.sm} !important;
+  opacity: 1 !important;
+  &::before,
+  &::after {
+    border: none !important;
+  }
+`;
```

Comment:
You  can add css property to wrapper and override library's classes + Please add box shadow